### DISCUSSION TITLE
AVR128DDxx Parts

### DISCUSSION THREAD

**Discussion by btribley** [SpenceKonde/DxCore#102] [Future development discussion]
Anyone have any informed rumors as to when these parts may be available? I've heard they will be reduced pin count chips suitable for systems not needing as much external I/O.  Really enjoying the AVR128DA28 DxCore, I want to standardize on this going forward for all my projects. Presently I am using PIC microcontrollers for simple stuff, getting really ,really tired of MPLAB-IDE and its never ending bug trail...

**Comment by SpenceKonde** [comment]
You will be waiting a very long time for a avr128dd- there is no indication
they have any plans for a 128k dd - series part, it is announced for 64k
32k and 16k parts in 14, 20 28 and 32 pin versions, starting with 64k which
was just added to the atpacks. See the avr dd product brief. I heard first
half of 2021, dont lnow anything beyond that, am trying to get as much as
possible ready so I can quickly add support when hardware is available

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***

**Reply by btribley** [reply]
Thanks, Spence.  64K is plenty for most of these applications.

**Comment by SpenceKonde** [comment]
It will be added to dxcore. I am very excited for these parts too (at least
the 14/20 pin version. Not sure the point of the 28/32 pin ones- but for
the errata on the db-series, which could be fixed st any time (mostly
thinking of the TCD alt pin problem) the db28/32 is strictly better)) I'm
laying out circuit boards for them to have on standby when they arrive, too.

____________
Spence Konde
Azzy’S Electronics

New products! Check them out at tindie.com/stores/DrAzzy
GitHub: github.com/SpenceKonde
ATTinyCore: Arduino support for almost every ATTiny microcontroller
Contact: ***@***.***

**Reply by SpenceKonde** [reply]
One bit of depressing news - it is looking like the 28 and 32 pin parts which are largely redundant will be coming first! (based on their having pages listing them as "future product" while the smaller ones that we're all drooling over don't....

**Reply by btribley** [reply]
They are already tooled up for that form factor. I agree with you completely that given the low cost of the 128K parts they are redundant. There are many designs that require smaller chips which is why the PIC line has never given up on the 14, 18 and 20 pin parts. Until they come out with those parts Arduino will not be able to infiltrate that space with advanced chip designs.
